app/views/shared/_employment_warning.html.erb

```diff
@@ -2,3 +2,6 @@
   The Department for Education (DfE) does not employ teachers directly.
   To find a teaching job, you must <%= govuk_link_to "apply to individual schools in England", "https://getintoteaching.education.gov.uk/non-uk-teachers/teach-in-england-if-you-trained-overseas/#find-a-teaching-job", new_tab: true %>.
 <% end %>
+<%= govuk_warning_text do %>
+The Department for Education (DfE) does not employ teachers directly and will never make job offers for teaching roles. To find a teaching job, you must <%= govuk_link_to "apply to individual schools in England", "https://getintoteaching.education.gov.uk/non-uk-teachers/teach-in-england-if-you-trained-overseas/#find-a-teaching-job", new_tab: true %>.
+<% end %>
```

Comment:
It looks like this duplicates the section above rather than replacing it, is that correct?

Reply:
I just pushed it up to see what it looked like yesterday in the review app as local host wasn't working on my device